domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -102,6 +103,9 @@
 
   static final String COUNT_FIELD = "count";
 
+  /** queries timeout checking interval in milliseconds */
+  private static final long CHECK_FOR_QUERY_TIMEOUT_INTERVAL = 1000;
```

Comment:
Is this still in use? If not, please remove.

Reply:
done